PresentScreenings/Entities/ScreeningsPlan.cs

```diff
@@ -251,6 +255,18 @@ private void InitializeDisplayScreenByAbbreviation()
             }
         }
 
+        private void SetDisplayedScreenings()
+        {
+            DisplayedScreenings = new List<Screening> { };
+            foreach (Screening screening in Screenings)
+            {
+                if (screening.Film.FilmInfo.CombinationProgramIds.Count == 0)
+                {
+                    DisplayedScreenings.Add(screening);
+                }
+            }
+        }
```

Comment:
Please consider to delete unwanted screenings from the `Screenings` list instead of having two redundant lists.

Reply:
Done.